package.json

```diff
@@ -97,6 +97,6 @@
   },
   "main": "qunit/qunit.js",
   "engines": {
-    "node": "4.* || 6.* || 7.*"
```

Comment:
I suspect we don't want to support unsupported versions.

`"node": "4.* || 6.* || 7.* || >= 8"`

note, 5 isn't in that list and shortly (end of this month, 7 will also be dropped)